scss/foundation.scss

```diff
@@ -69,8 +69,13 @@
 
 @mixin foundation-everything(
   $flex: true,
-  $prototype: false
+  $prototype: false,
+  $xy-grid: true
```

Comment:
In this case I think we should use:
```scss
$xy-grid: $xy-grid
```
for backward compatibility. The global `$xy-grid` setting should still have an effect.
Please check it works as expected.

Reply:
Ok, will check this later today before pushing this proposed change.